### PR TITLE
feat(components/breadcrumb): refresh style

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
[TCCUI](https://jira.talendforge.org/browse/TDP-6895) related changes 

**What is the chosen solution to this problem?**
Refresh breadcrumb style

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR
https://company-57688.frontify.com/document/92132#/navigation-layout/breadcrumb-1555666347

**[x] This PR introduces a breaking change**

**Before** we used `maxItems ` to specify how many items should be always displayed after the ellipsis button.

| maxItems | example |
|--- | --- |
| 2 | `Home / Folder 1` |
| 2 | `… / Folder 1 / Folder 2` |
| 3 | `Home / Folder 1 / Folder 2` |
| 3 | `… / Folder 2 / Folder 3 / Folder 4` |

**Now**, root item is always visible, so we use `maxItems` to specify how many items should be displayed including root and the ellipsis button

| maxItems | example |
|--- | --- |
| 3 | `Home / Folder 1 / Folder 2` |
| 3 | `Home / … / Folder 3` |
| 4 | `Home / … / Folder 3 / Folder 4` |
| 4 | `Home / … / Folder 4 / Folder 5` |

So if you used
```jsx
<Breadcrumb maxItems='2'  />
``` 
you must consider changing it to 
```jsx
<Breadcrumb maxItems='4'  />
```

### CODE REVIEW

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js [hunk 1]
```diff
@@ -6,35 +6,64 @@ import { translate } from 'react-i18next';
...
+			<div
+				className={classNames(
+					theme['tc-breadcrumb'],
+					theme.loading,
+					'tc-breadcrumb',
+					'tc-breadcrumb--loading',
+				)}
+			>
+				{[
+					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
```
Comment (jare-talend) [inline packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: I prefer use a constant to set all skeleton and map on this for lisibility.
```suggestion
const BREADCRUMB_SKELETONS = [
					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
					{ size: Skeleton.SIZES.small, type: Skeleton.TYPES.circle },
					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
				]
				
				BREADCRUMB_SKELETONS.map...

```

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js [hunk 2]
```diff
@@ -6,35 +6,64 @@ import { translate } from 'react-i18next';
...
+				].map(({ size, type }) => (
+					<Skeleton size={size} type={type} />
+				))}
+			</div>
+		);
+	}
+	const nbItems = items.length;
+	const maxItemsReached = nbItems > maxItems;
+	const ellipsisIndex = nbItems - 1 - maxItems + 1 + DEFAULT_NB_ITEMS_BEFORE_ELLIPSIS;
```
Comment (jare-talend) [inline packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: -1+1 = 0

```suggestion
	const ellipsisIndex = nbItems - maxItems + DEFAULT_NB_ITEMS_BEFORE_ELLIPSIS;
```

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js [hunk 3]
```diff
@@ -6,35 +6,64 @@ import { translate } from 'react-i18next';
...
+					'tc-breadcrumb',
+					'tc-breadcrumb--loading',
+				)}
+			>
+				{[
+					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
+					{ size: Skeleton.SIZES.small, type: Skeleton.TYPES.circle },
+					{ size: Skeleton.SIZES.large, type: Skeleton.TYPES.text },
+				].map(({ size, type }) => (
+					<Skeleton size={size} type={type} />
```
Comment (jmfrancois) [inline packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: map need a key

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js (hunk 2 above)
Comment (jmfrancois) [reply packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: la tete a toto

Comment (ncomont) [inline packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: `const ellipsisIndex = nbItems - maxItems + DEFAULT_NB_ITEMS_BEFORE_ELLIPSIS;`

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js [hunk 4]
```diff
@@ -83,65 +103,63 @@ export function BreadcrumbsComponent(props) {
...
 		if (maxItemsReached && index === ellipsisIndex) {
-			return [
+			return (
 				<li className="tc-breadcrumb-menu" key={index + 0.1}>
 					<ActionDropdown
-						id={`${props.id}-ellipsis`}
+						id={`${id}-ellipsis`}
 						items={hiddenItems}
-						aria-label={props.t('BREADCRUMB_OPEN_FIRST_LINKS_MENU', {
+						aria-label={t('BREADCRUMB_OPEN_FIRST_LINKS_MENU', {
```
[the remark this answers is not in the document]
Comment (frassinier) [reply packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: Why not? 🤔

**Pull request merged**